### PR TITLE
revise def and xrefs for tooth agenesis, selective, 1 

### LINKED ISSUE #7034: MSX1-related tooth agenesis with or without orofacial cleft
**Preferred gene-related syndrome label**
MSX1-related tooth agenesis with or without orofacial cleft 

**Synonyms**

**Parent term (use [OLS](https://www.ebi.ac.uk/ols/ontologies/mondo), or your favorite ontology browser)**

**Definition**
 A common developmental disease characterized by varying severity of tooth agenesis that may be seen in combination with orofacial clefting in some individuals 

**Definition source (Please give PubMed ID, if applicable, in format PMID:#######)
PMID: 27699475, 35877402

**Children terms (if applicable)** Should any existing terms be re-classified as children underneath this new proposed term? 

**Your nano-attribution (ORCID) or URL for a working group**
https://clinicalgenome.org/affiliation/40059/

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
and split out 'hypodontia/oligodontia with orofacial cleft' and 'tooth agenesis, selective, with orofacial cleft' as separate terms and removed these as synonyms from 'tooth agenesis, selective, 1'

close #7034

### CODE REVIEW

**Review by twhetzel — APPROVED** [review]
looks good. was this information from the ticket supposed to be an annotation?
Preferred gene-related syndrome label
MSX1-related tooth agenesis with or without orofacial cleft

**Comment by nicolevasilevsky** [comment]
> looks good. was this information from the ticket supposed to be an annotation? Preferred gene-related syndrome label MSX1-related tooth agenesis with or without orofacial cleft

good catch, I added this

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -134098,21 +134098,20 @@ is_a: MONDO:0003847 {source="MESH:C562461/inferred"} ! hereditary disease
...
 name: tooth agenesis, selective, 1
-def: "Any tooth agenesis in which the cause of the disease is a mutation in the MSX1 gene." [MONDO:patterns/disease_series_by_gene]
+def: "Any tooth agenesis in which the cause of the disease is a mutation in the MSX1 gene characterized by varying severity of tooth agenesis that may be seen in combination with orofacial clefting in some individuals." [https://clinicalgenome.org/affiliation/40059/, MONDO:patterns/disease_series_by_gene, PMID:27699475, PMID:35877402]
 subset: gard_rare {source="GARD:18244"}
 subset: nord_rare {source="MONDO:NORD"}
 subset: rare
 synonym: "hypodontia/oligodontia 1" RELATED [OMIM:106600]
-synonym: "hypodontia/oligodontia with orofacial cleft" RELATED [OMIM:106600]
 synonym: "MSX1 tooth agenesis" EXACT [MONDO:design_pattern, MONDO:patterns/disease_series_by_gene]
+synonym: "MSX1-related tooth agenesis with or without orofacial cleft" EXACT [https://clinicalgenome.org/affiliation/40059/]
```
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:134107]: Thanks! I gather that adding it as a synonym is what's needed for "Preferred gene-related syndrome label".

**Review by twhetzel — APPROVED** [review]
approved, but added one comment to clarify the new content was intended as a `synonym`.

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:134107]: yes, they asked for the new term with a new label, but the term already existed, so I added their label as a synonym.

**Pull request merged**